### PR TITLE
Form Page Enhancements

### LINKED ISSUE #120: Implement Form Page Enhancements: Balance Cards, Background & Styling Updates, and Alert Positioning
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Add balance cards to Form page according to figma with functionality for interacting with the server to receive data.
3. Add background styles to Form page: stars and background gradient, according to figma.
4. Add styles to radio buttons "Select Token" and "Select Multiplier" at the Form page, according to figma.
5. Move alert message "Please fill the form" above radio buttons "Select Token" according to figma.
6. Text me to telegram, I will give you access to Figma if you need it (my telegram handler: djeck_vorobey1)

### PR DESCRIPTION
**Pull request by BrunoAmbricca**
closes #120 

Tasks:
1.Done
2.Done
3.Done
4.Done (Please check STRK radio button if the svg is ok when checked)
5.Done
6.Notified

### CODE REVIEW

File: frontend/src/pages/forms/form.css [hunk 1]
```diff
@@ -299,4 +310,81 @@ input[type="number"].error {
...
+
+.balance-item label:nth-child(1){
+    font-size: 20px;
+    color: #E7ECF0;
+    letter-spacing: 0.5px;
+}
+
+.balance-item label:nth-child(2){
+    font-size: 32px;
+    color: #E7ECF0;
```
Comment (whateverfw) [inline frontend/src/pages/forms/form.css:378]: use css variable

File: frontend/src/pages/forms/form.css [hunk 2]
```diff
@@ -299,4 +310,81 @@ input[type="number"].error {
...
+
+.balance-title{
+    display: flex;
+    align-items: center;
+}
+
+.balance-title svg{
+    margin-right: 5px;
+    height: 25px !important;
+    width: 24px !important;
```
Comment (whateverfw) [inline frontend/src/pages/forms/form.css:389]: please avoid important

File: frontend/src/pages/forms/form.css [hunk 3]
```diff
@@ -299,4 +310,81 @@ input[type="number"].error {
...
+    border: 0.5px solid #4e7787;
+    justify-content: center;
+    background: linear-gradient(150deg, #74D6FD80 1.13%, #0B0C1080 103.45%);
+    margin-right: 20px;
+    box-shadow: 3.75px 3.75px 9.38px 0px #99EAFF40 inset;
+}
+
+.balance-item label:nth-child(1){
+    font-size: 20px;
+    color: #E7ECF0;
```
Comment (whateverfw) [inline frontend/src/pages/forms/form.css:372]: use css variable

File: frontend/src/pages/forms/Form.jsx [hunk 4]
```diff
@@ -91,6 +97,19 @@ const Form = ({ walletId, setWalletId }) => {
                     <div className="submit">
                         <button type="submit" className='form-button'>Submit</button>
                     </div>
+                    <div className="card-gradients forms-gradient">
+                        <div className="card-gradient"></div>
+                        <div className="card-gradient"></div>
+                    </div>
```
Comment (whateverfw) [inline frontend/src/pages/forms/Form.jsx:103]: create a separate component called CardGradients, that accepts className prop, which is used alongside with "card-gradients", (seems like similar code is used in other places, the only different is this second className, in this case it is `forms-gradient`)

File: frontend/src/pages/forms/Form.jsx [hunk 5]
```diff
@@ -91,6 +97,19 @@ const Form = ({ walletId, setWalletId }) => {
...
+                    {starData.map((star, index) => (
+                    <Star key={index} style={{
+                        position: 'absolute',
+                        top: `${star.top}%`,
+                        left: `${star.left}%`,
+                        width: `${star.size}%`,
+                        height: `${star.size}%`
+                        }}/>
+                    ))}
```
Comment (whateverfw) [inline frontend/src/pages/forms/Form.jsx:112]: This should be a separate component, that accept starData as a prop

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/pages/forms/Form.jsx (hunk 4 above)
Comment (BrunoAmbricca) [reply frontend/src/pages/forms/Form.jsx]: Please check if I understood correctly.

Comment (whateverfw) [reply frontend/src/pages/forms/Form.jsx]: yes, thx

**Review by whateverfw — APPROVED** [review]

**Review by djeck1432 — APPROVED** [review]
Looks good to me

**Pull request merged**